misc/bstr.h

```diff
@@ -150,6 +150,8 @@ struct bstr bstr_strip_linebreaks(struct bstr str);
  */
 void bstr_xappend(void *talloc_ctx, bstr *s, bstr append);
 
+#define bstr_xappend0(ctx, dst, s) bstr_xappend(ctx, dst, bstr0(s))
```

Comment:
Need parentheses around `ctx` and `dst`. Also generally we define it as inline functions in this header.